operator/cmd/mesh/profile-diff.go

```diff
@@ -51,6 +51,7 @@ func profileDiffCmd(rootArgs *rootArgs, pfArgs *profileDiffArgs) *cobra.Command
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SetOut(os.Stdout)
 			return profileDiff(cmd, rootArgs, pfArgs, args)
```

Comment:
Now that I think about it, maybe we should have `profileDiff` take an `io.Writer` and pass `cmd.OutOrStdout`?